packer/resource-ami/packer-conf.json

```diff
@@ -0,0 +1,50 @@
+{
+    "variables": {
+        "aws_access_key": "{{env `AWS_ACCESS_KEY_ID`}}",
+        "aws_secret_key": "{{env `AWS_SECRET_ACCESS_KEY`}}",
+	"source_ami_id":  "{{env `PACKER_SOURCE_AMI_ID`}}",
+	"os": "{{env `PACKER_SOURCE_OS`}}",
+	"os_version": "{{env `PACKER_SOURCE_OS_VERSION`}}",
+	"ssh_user_name": "{{env `PACKER_SSH_USERNAME`}}",
+	"ami_creator_ip": "{{env `PACKER_AMI_CREATOR_IP`}}",
+        "region": "us-east-1"
+    },
+    "builders": [
+        {
+            "access_key": "{{user `aws_access_key`}}",
+            "ami_name": "TestGrid-{{user `os`}}-{{user `os_version`}}-{{isotime \"2006-01-02\"}}-{{timestamp}}",
+            "instance_type": "t2.micro",
+            "region": "us-east-1",
+            "secret_key": "{{user `aws_secret_key`}}",
+            "source_ami": "{{user `source_ami_id`}}",
+            "ssh_username": "{{user `ssh_user_name`}}",
+            "type": "amazon-ebs",
+	    "run_tags" : {
+                "OS" : "{{user `os`}}",
+                "Tool" : "Packer"
+            },
+	    "tags": {
+        	"OS": "{{user `os`}}",
+		"OSVersion": "{{user `os_version`}}",
+		"Creator": "{{user `ami_creator_ip`}}",
+        	"Timestamp": "{{timestamp}}",
+		"AGENT_READY": "true"
```

Comment:
We tried to get rid of this, yes?

Reply:
Yes we will. This will get rid once with the relevant changes in TestGrid repository is merged. (PR is to be added there)